scalekit/mcp.py

```
@@ -1,4 +1,4 @@
-from typing import Optional, List, Dict, Any
+from typing import Optional, List
 
 from scalekit.core import CoreClient
 from scalekit.v1.mcp.mcp_pb2 import *
```

Ruff (0.14.2)

`from scalekit.v1.mcp.mcp_pb2 import *` used; unable to detect undefined names

(F403)